### ISSUE TITLE
Conbee 3 support?

### ISSUE THREAD

**Issue by xxLeoxx93** [zigpy/zigpy-deconz#233]
Just got my ConBee 3 but it seems like it's not supported yet. Is that correct or should it work already?
Thanks!

**Comment by puddly** [comment]
It'll be in the next HA beta: https://github.com/zigpy/zigpy-deconz/pull/232

**Comment by xxLeoxx93** [comment]
Alright, I'll probably switch to beta once it's available. Thanks for the quick response!

**Comment by pipiche38** [comment]
@puddly for the support of ConBee 3, do we have just to adapt the serial speed, or is there any other things to care ?

**Comment by puddly** [comment]
@pipiche38 It'll work the same as any other radio. Only thing currently broken is energy scanning.

**Comment by xxLeoxx93** [comment]
> It'll be in the next HA beta: #232

Sorry that I ask again but is this now part of 12.2023 release? Or still just in the beta of HA?

**Comment by MattWestb** [comment]
Its released https://www.home-assistant.io/blog/2023/12/06/release-202312/

**Comment by xxLeoxx93** [comment]
@MattWestb 
Well, the new HA version is released but it seems like Conbee 3 Support isn't part of it yet.
So I assume only the beta branch of HA has this implemented yet?

**Comment by puddly** [comment]
It's part of the release. Does it not work for you?

**Comment by xxLeoxx93** [comment]
No. Im on Docker, Core version 2023.12.
This is what I got when I plug in the conbee 3 stick:
![image](https://github.com/zigpy/zigpy-deconz/assets/61089790/5b6b81d6-ea8e-45bb-92ad-0341a9e77248)
afterwards in the selection menue I don't see conbee 3 either:
![image](https://github.com/zigpy/zigpy-deconz/assets/61089790/3cc34443-dcf6-4cd3-86f7-f7e0ed2f704c)
(Sorry for the german texts, if anything needs translation let me know)

however looking in my system directly it identifies the conbee 2 correctly but not the conbee 3 (?)
![image](https://github.com/zigpy/zigpy-deconz/assets/61089790/9fdbe7b9-8baf-4dd4-ae7b-dae0f70190d4)
(First section is the conbee 3, second the conbee 2 attached) In windows conbee 3 is detected by deconz instantly.

**Comment by puddly** [comment]
That sounds like a configuration problem with your Docker container. It shows up just fine here:

```
$ dmesg
...
[1640352.437118] usb 1-3.1.1.1: new full-speed USB device number 20 using ehci-pci
[1640352.555003] usb 1-3.1.1.1: New USB device found, idVendor=0403, idProduct=6015, bcdDevice=10.00
[1640352.555011] usb 1-3.1.1.1: New USB device strings: Mfr=1, Product=2, SerialNumber=3
[1640352.555015] usb 1-3.1.1.1: Product: ConBee III
[1640352.555019] usb 1-3.1.1.1: Manufacturer: dresden elektronik
[1640352.555023] usb 1-3.1.1.1: SerialNumber: DE03111373
[1640352.557668] ftdi_sio 1-3.1.1.1:1.0: FTDI USB Serial Device converter detected
[1640352.557779] usb 1-3.1.1.1: Detected FT-X
[1640352.558876] usb 1-3.1.1.1: FTDI USB Serial Device converter now attached to ttyUSB1
```

```
$ lsusb -v
...
Bus 001 Device 020: ID 0403:6015 Future Technology Devices International, Ltd Bridge(I2C/SPI/UART/FIFO)
Device Descriptor:
  bLength                18
  bDescriptorType         1
  bcdUSB               2.00
  bDeviceClass            0
  bDeviceSubClass         0
  bDeviceProtocol         0
  bMaxPacketSize0         8
  idVendor           0x0403 Future Technology Devices International, Ltd
  idProduct          0x6015 Bridge(I2C/SPI/UART/FIFO)
  bcdDevice           10.00
  iManufacturer           1 dresden elektronik
  iProduct                2 ConBee III
  iSerial                 3 DE03111373
  bNumConfigurations      1
  Configuration Descriptor:
    bLength                 9
    bDescriptorType         2
    wTotalLength       0x0020
    bNumInterfaces          1
    bConfigurationValue     1
    iConfiguration          0
    bmAttributes         0x80
      (Bus Powered)
    MaxPower               90mA
    Interface Descriptor:
      bLength                 9
      bDescriptorType         4
      bInterfaceNumber        0
      bAlternateSetting       0
      bNumEndpoints           2
      bInterfaceClass       255 Vendor Specific Class
      bInterfaceSubClass    255 Vendor Specific Subclass
      bInterfaceProtocol    255 Vendor Specific Protocol
      iInterface              2 ConBee III
      Endpoint Descriptor:
        bLength                 7
        bDescriptorType         5
        bEndpointAddress     0x81  EP 1 IN
        bmAttributes            2
          Transfer Type            Bulk
          Synch Type               None
          Usage Type               Data
        wMaxPacketSize     0x0040  1x 64 bytes
        bInterval               0
      Endpoint Descriptor:
        bLength                 7
        bDescriptorType         5
        bEndpointAddress     0x02  EP 2 OUT
        bmAttributes            2
          Transfer Type            Bulk
          Synch Type               None
          Usage Type               Data
        wMaxPacketSize     0x0040  1x 64 bytes
        bInterval               0
Device Status:     0x0000
  (Bus Powered)
```

**Comment by xxLeoxx93** [comment]
So, I managed to make it work (somehow a reboot fixed it) and it detected the conbee 3 just fine now.
Unfortunatly the migation doesn't work at all. Neither using the automated backup nor a manual one works and just shows "unknown error". 
I've attached my logfile if that helps.
[home-assistant_2023-12-08T19-40-12.117Z.log](https://github.com/zigpy/zigpy-deconz/files/13620287/home-assistant_2023-12-08T19-40-12.117Z.log)

Thanks a lot for your support!
